smt/utils/kriging_utils.py

```diff
@@ -20,11 +20,13 @@
 # TODO: Create hyperclass Kernels and a class for each kernel
 
 
-def standardization(X, y, copy=False):
+def standardization(X, y, copy=False, is_Unit_X=False):
 
     """
+
     We substract the mean from each variable. Then, we divide the values of each
-    variable by its standard deviation.
+    variable by its standard deviation. If isUnitX, we scale the input space X to
```

Comment:
Should be <code>is_unit_X</code> but I wonder whether it is the right name. When you call the method you ask for an action (like copy), starting with an action verb would be better, may be something like <code>scale_X_to_unit</code>?